internal/commands/build.go

```diff
@@ -44,6 +46,7 @@ type BuildFlags struct {
 	GID                int
 	PreviousImage      string
 	SBOMDestinationDir string
+	DateTime           string
```

Comment:
This should probably match the flag a little bit better. `DateTime` is a little ambiguous.

```suggestion
	CreationTime           string
```

Reply:
Agree with @jromero , I think `DateTime` would be confusing in this context, where there could conceivably be multiple times. Being explicit that this is `CreationTime` would be great

Reply:
> Isn't this variable only respected on newer Platform API versions?

This is interesting... I had a different memory from the [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0103-source-date-epoch.md) - I thought that while it would be spec'd in 0.9 (so that platforms can look in the spec and know how to use it), it would be respected in all platform api versions. However it looks like the [lifecycle implementation](https://github.com/buildpacks/lifecycle/blob/b6803be364429689a818a5d0a09db42bd21b9995/cmd/lifecycle/exporter.go#L442-L444) says otherwise. I'll change it.